support-frontend/assets/helpers/forms/paymentIntegrations/payPalRecurringCheckout.ts

```diff
@@ -221,21 +225,24 @@ function setupPayment(
 	};
 }
 
-function getPayPalEnvironment(isTestUser: boolean): string {
-	return isTestUser
+const getPayPalEnvironment = (isTestUser: boolean): string =>
```

Comment:
There's a mix of function declarations and expressions here, should we settle on one approach for consistency, I believe within the team there was a consensus to go with declarations?

Reply:
Did we decide on function declarations for react components and constants for non react components?